src/Altinn.Platform/Altinn.Platform.PDF/Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM maven:3.6.3-jdk-14 AS build
+FROM maven:3.6.3-openjdk-15 AS build
 WORKDIR build
 COPY . .
 RUN mvn -Pprod package
 
-FROM openjdk:16-jdk-alpine3.12 AS final
+FROM openjdk:15-jdk-alpine3.12 AS final
```

Comment:
Downgrade? 
If 14 and 16 could run together couldt 15 and 16 do the same?

Reply:
Yes, but 16 is a prerelease. We have been running on 16 for a while, so it would probably be fine, but the recommendation is to stay on 15 for production systems.

Reply:
There was a reason for that. Not sure if it's still relevant. https://jdk.java.net/15/
![image](https://user-images.githubusercontent.com/6088624/103902722-37e99380-50fb-11eb-9f14-154abe85deb3.png)

Reply:
I that case I picking the easiest solution and keep 16. We know it works.